common/src/test/resources/logback.xml

```diff
@@ -40,14 +38,14 @@
   -->
   <logger name="org.apache.drill" additivity="false">
     <level value="debug" />
-    <appender-ref ref="SOCKET" />
+<!--    <appender-ref ref="SOCKET" /> -->
```

Comment:
the socket appender adds 1 second delay in the startup. That's why I turned it off by default.
We should review those files has there is always more than one on the class path and we have warnings about only using the first one that is found.
There is a notion of -test log config that is not used.
possibly the socket appender could be turned on with a env variable or some setting?